src/Module.php

```diff
@@ -53,7 +55,9 @@ public function onBootstrap(MvcEvent $e)
      */
     public function onRender(MvcEvent $e)
     {
-        $app      = $e->getTarget();
+        $app = $e->getTarget();
+
+        /** @psalm-suppress PossiblyNullReference */
```

Comment:
Shouldn't be suppressed: should expand baseline instead

Reply:
Fixed